requirements/pytorch/base.txt

```diff
@@ -11,3 +11,4 @@ torchmetrics>=0.7.0, <0.9.3  # needed for using fixed compare_version
 pyDeprecate>=0.3.1, <=0.3.2
 packaging>=17.0, <=21.3
 typing-extensions>=4.0.0, <4.3.1
+pl_devtools>=0.2.0
```

Comment:
```suggestion
```
development tools shall not be in "production" package

Reply:
The package does not just contain development tools. Maybe what's wrong is the package name